specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2018-11-19/machineLearningServices.json

```diff
@@ -835,6 +835,55 @@
       }
     }
   },
+  "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/computes/{computeName}/addNodeUser": {
+    "post": {
```

Comment:
> "post":  [](start = 3, length = 9)

note that this operation (adding user) can't be represented in ARM template. (as this is  POST)